pinot-common/src/main/java/org/apache/pinot/common/restlet/resources/SegmentErrorInfo.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.common.restlet.resources;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+
+
+/**
+ * This class is used to represent errors related to a segment.
+ */
+@SuppressWarnings("unused")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SegmentErrorInfo {
+  private static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss z";
+  private static final SimpleDateFormat SIMPLE_DATE_FORMAT = new SimpleDateFormat(DATE_FORMAT, Locale.getDefault());
+
+  private final String _timestamp;
+  private final String _errorMessage;
+  private final String _exceptionStackTrace;
+
+  /**
+   *
+   * @param exceptionStackTrace Exception object
+   * @param errorMessage Error Message
+   * @param timestampMs Timestamp of the error
+   */
+  public SegmentErrorInfo(Exception exceptionStackTrace, String errorMessage, long timestampMs) {
```

Comment:
```suggestion
  public SegmentErrorInfo(Exception exception, String errorMessage, long timestampMs) {
```

Reply:
(nit) change the parameter order to match the variable order

Reply:
In previous review, you suggested to rename `exception` to `stackTrace`, I used `exceptionStackTrace`. Do you want to go back to `exception`? I feel using `stackTrace` or `exceptionStackTrace` is explicit.

Reply:
This one is still the `Exception` instead of the stack trace. Once you write the stack trace as a string, then name it `exceptionStackTrace`